internals/review_models.py

```diff
@@ -332,11 +337,14 @@ class Activity(ndb.Model):  # copy from Comment
   amendments = ndb.StructuredProperty(Amendment, repeated=True)
 
   @classmethod
-  def get_activities(cls, feature_id, gate_id=None):
+  def get_activities(cls, feature_id: int, gate_id: Optional[int]=None,
+      comments_only: bool=False) -> list[Activity]:
     """Return actitivies for an approval."""
     query = Activity.query().order(Activity.created)
     query = query.filter(Activity.feature_id == feature_id)
     if gate_id:
       query = query.filter(Activity.gate_id == gate_id)
     acts = query.fetch(None)
+    if comments_only:
+      return [act for act in acts if act.content]
```

Comment:
@jrobbins I wasn't sure of a better way to do this. We currently have nothing to discern comment entities from Activities that represent changes to the feature. Is there a better way to do this? Or would there be merit in adding an `is_comment` field or something similar?

Reply:
Checking for content is fine.